hi-ml-histopathology/testhisto/testhisto/mocks/base_data_generator.py

```diff
@@ -59,10 +60,11 @@ def __init__(
         tile_size: int = 28,
     ) -> None:
         """
-        :param tmp_path: A temporary directory to store all generated data.
         :param mock_type: The wsi generator mock type. Supported mock types are:
+        :param tmp_path: A temporary directory to store all generated data.
```

Comment:
Can we rename tmp_path to distinguish it from the pytest fixture?

Reply:
Sure!